gpu4pyscf/drivers/dft_driver.py

```diff
@@ -228,9 +238,24 @@ def run_dft(mol_name, config, charge=None, spin=0):
             start_time = time.time()
             h = mf.Hessian()
```

Comment:
Consider to use `with` context to wrap the h5py file, or flushing the h5py file before executing the heavy computation.